projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/no-linked-cases/no-linked-cases.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { LinkedCasesService } from '../../services/linked-cases.service';
+
+@Component({
+  selector: 'ccd-no-linked-cases',
+  templateUrl: './no-linked-cases.component.html'
+})
+export class NoLinkedCasesComponent {
+
+  constructor(private readonly router: Router,
+    public linkedCasesService: LinkedCasesService) {
```

Comment:
@johnbenjamin-hmcts - query `public` vs `private`

Reply:
@Daniel-Lam I have updated the code now.